src/Button.tsx

```diff
@@ -133,7 +301,8 @@ export const Button: React.FC<Props> = ({
             ? 112
             : 100,
 
-          padding: size === "small" ? `5px 8px` : size === "large" ? 8 : 7,
+          paddingLeft: paddingY,
```

Comment:
Shouldn't these be called `paddingX` since they're on the left and right side?

Reply:
Haha yes 🤕